backend/src/main/resources/application.properties

```
@@ -5,3 +5,8 @@ spring.datasource.url=jdbc:postgresql:${DB_HOST://localhost:5432/compass_db}
 spring.datasource.username=${DB_USERNAME}
 spring.datasource.password=${DB_PASSWORD}
 spring.jpa.hibernate.ddl-auto=${DDL_AUTO}
+
+cors.allowedOrigins=http://localhost:3000,https://compass-2.pm4.init-lab.ch
```

```suggestion
cors.allowedOrigins=http://localhost:3000,https://${APP_URL}
```
Weiss nicht ob `https://compass-2.pm4.init-lab.ch` hier notwendig ist weil jetzt ist die domain `https://compass.pm4.init-lab.ch` und es funktioniert.